components/Toster/Toaster.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { resolveValue, Toaster, ToastBar } from 'react-hot-toast';
+import { useTheme } from 'next-themes';
+
+export const AppToaster = () => {
+  const { theme } = useTheme();
+
+  return (
+    <Toaster position="top-center" toastOptions={{ className: 'app-toaster' }}>
```

Comment:
Add the text color for each mode. Do test it

Reply:
![image](https://user-images.githubusercontent.com/14018471/194010658-e28f60ff-4ba7-4b1c-b03f-baf857a258f1.png)


> @titusjoyson, I saw success toast was the text was in black color in dark mode. which was not visible

Can you share a screenshot, I was able to see white text in black mode.

Reply:
Found it in chrome its not working 
![image](https://user-images.githubusercontent.com/14018471/194015612-594f5f88-cc6e-420f-a541-c3d1b6ac92dc.png)

Reply:
yes, i had this issue previously.

Reply:
It seems next-themes not setting the theme properly or setting it to system, which was causing this issue. Made some changes to ThemeToggler to have a reusable implementation of isDark variable using hooks. Hava a look
@rohitdasu @leewan-09 - Hope fix works for you guys as well.

Reply:
it worked for me